integration_tests/__tests__/test_path_pattern_reporter_message.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+import runJest from '../runJest';
+import {cleanup, writeFiles} from '../utils';
+import os from 'os';
+import path from 'path';
+
+const skipOnWindows = require('skipOnWindows');
+const DIR = path.resolve(os.tmpdir(), 'jest_path_pattern_reporter_message');
+
+skipOnWindows.suite();
+
+beforeEach(() => cleanup(DIR));
+afterEach(() => cleanup(DIR));
+
+test('prints a message with path pattern at the end', () => {
+  writeFiles(DIR, {
+    '.watchmanconfig': '',
+    '__tests__/a.test.js': `test('a', () => {});`,
+    '__tests__/b.test.js': `test('b', () => {});`,
+    'package.json': '{}',
+  });
+  let stderr;
+
+  ({stderr} = runJest(DIR, ['a']));
+  expect(stderr).toMatch('Ran all test suites matching "a"');
```

Comment:
Maybe there was some idea behind this?  But I like showing regex here. To be even more precise, we could change the message to:
```
Ran all test suites matching pattern /some.*regex/
```
Similar to what we do in watch mode.